### DISCUSSION TITLE
authorization token

### DISCUSSION THREAD

**Discussion by mmagithub** [polaris-hub/polaris#232] [Q&A]
Hi

I recently registered for the repository, but I’m still unable to retrieve the authorization token required to access the datasets. Could you please let me know the steps to obtain it?

Thank you for your help.

Best regards,
Marawan

**Comment by Andrewq11** [comment]
Hi @mmagithub,

Thanks for reaching out. Can you expand on what you mean by "registered for the repository"? 

Also, can you confirm that you've done the following:
- Created an account on https://polarishub.io/
- Downloaded the Polaris Python client at https://pypi.org/project/polaris-lib/
- Ran `polaris login` in your terminal after installing the package

If you've done the above, the client should take care of handling the authorization token for you. [Here is a link](https://polaris-hub.github.io/polaris/stable/tutorials/basics.html) to a tutorial where the first few steps may be of use to you.

Looking forward to hearing back.
